buildtapas.make

```diff
@@ -56,7 +57,7 @@ projects[diff][subdir] = develop
 projects[tapas-modules][type] = module
 projects[tapas-modules][download][type] = git
 projects[tapas-modules][download][url] = git://github.com/NEU-DSG/tapas-modules
-projects[tapas-modules][download][branch] = develop
+projects[tapas-modules][download][branch] = feature/develop
```

Comment:
@lindajeanne This has a typo in it.  The branch should be develop, not feature/develop.